envs/differential-stats.yaml

```diff
@@ -6,6 +6,7 @@ dependencies:
  - perl-atlas-modules >=0.3.1
  - r-atlas-internal>=1.8
  - r-ggplot2
+ - r-reshape2
```

Comment:
Just a note for as and when this starts being used with containers- there is already a container for this built from https://github.com/ebi-gene-expression-group/atlas-containers/tree/master/atlas-analysis-base

Reply:
Thanks @pinin4fjords  I have added a note. Not sure at the moment how the version restriction will `affect perl-atlas-modules` and `r-atlas-internal`, but it would be good to remove those restrictions.